kernel/aster-nix/src/util/net/addr.rs

```diff
@@ -207,6 +213,34 @@ pub struct CSocketAddrInet6 {
     sin6_scope_id: u32,
 }
 
+/// vm socket address
+#[repr(C)]
+#[derive(Debug, Clone, Copy, Pod)]
+pub struct CSocketAddrVm {
+    /// always [SaFamily::AF_VSOCK]
+    svm_family: u16,
+    /// always 0
+    svm_reserved1: u16,
+    /// Port number in host byte order.
+    svm_port: u32,
```

Comment:
Because Asterinas is built on host, `u32` should be aligned with the host byte order. Do we really need to explicitly use methods like `u32::from_le_bytes`?

Reply:
Actually, we don't want the `u32::from_le_bytes`. We can use `from_bytes` in `Pod` trait. 

But it may not be a big problem, u32 is also acceptable.